### ISSUE TITLE
Dear, The People

### ISSUE THREAD

**Issue by Crunchyyy456** [luckydog7/Funkin-android#1086]
**stop requesting mods you fuck tards, HEIS NOT IN CHARGED WITH MODS GO SEARCH ON YOUTUBE FOR THESE MODS YOU DIPSHITS**

Let the man rest you ungrateful dumbasses "plEaaaSe mr.deveeeelepor PllleeeaSEE PoRt WhiTtty buT wiTh BluE HeaD and CoolL mUsic" **shut the fuck up**

**Comment by ArcidonYT** [comment]
he do those cuz people are bored and the source code still not released thats why

**Comment by sonicyt1234** [comment]
SHUT UP YOU ANGRY LOSER, IF PEOPLE DON'T HAVE A PC, LUCKYDOG IS OKAY WITH PORTING MODS, ALSO, THE UPDATES OF THIS PORT ARE BASED ON THE *SOURCE CODE*
AND THE LAST UPDATE OF THE SOURCE CODE WAS BACK IN FEBRUARY, NOW SHUT THE FUCK UP

**Comment by lilpichu** [comment]
fuck you, you can't play calm because you eat, live, and work in mods, do you think on luckydog life? ARE YOU STUPID? LET'S LUCKYDOG TAKE A BREAK AND STOP CLAIMING MODS YOU CRAP, with that luckydog can take a break and work in week 7 =).

**Comment by lilpichu** [comment]
That's right, let's do the fight "luckydog rest a while". STOP REQUESTING MODS YOU CRAP!!

**Comment by Zakeblue1** [comment]
look, I know you are mad just because, people are asking for ports or something. i know how you feel about that but can you calm down?

**Comment by DabKnight** [comment]
I think this guy is the kod who raged from week 7 lol

**Comment by DzGithub2805** [comment]
> I think this guy is the kid who raged for week 7 lol

Maybe your fucking right

**Comment by fuckinstupid** [comment]
As it seems, Luckydog is porting mods because they're bored. The source code for week 7 hasn't been provided yet, so it's something they can do to occupy the time.

**Comment by EricizionS** [comment]
> As it seems, Luckydog is porting mods because they're bored. The source code for week 7 hasn't been provided yet, so it's something they can do to occupy the time.

true
